src/index.js

```diff
@@ -0,0 +1,4 @@
+require('esn-frontend-common-libs/src/frontend/vendor-libs.js');
```

Comment:
This is already required in the `esn-frontend-account` SPA, so it means, we load it two times.